cli/cmd/proxy/read/command.go

```diff
@@ -363,12 +404,7 @@ func (c *ReadCommand) outputRoutesTable(routes []Route) {
 	}
 
 	c.UI.Output(fmt.Sprintf("Routes (%d)", len(routes)), terminal.WithHeaderStyle())
-	table := terminal.NewTable("Name", "Destination Cluster", "Last Updated")
-	for _, route := range routes {
-		table.AddRow([]string{route.Name, route.DestinationCluster, route.LastUpdated}, []string{})
-	}
-	c.UI.Table(table)
-	c.UI.Output("")
+	c.UI.Table(formatRoutes(routes))
```

Comment:
It feels like the entire `outputRoutesTable` and `formatRoutes` can be one large method. There isn't behavior that is in here except the printing of headers which I think can be moved into the other file and maintained under a single method. This extends to the secrets, listeners endpoints and clusters as well. I think we should aim to have lesser indirection.

Reply:
I like to avoid indirection too. In this case, however, I like the testability of the format functions for now, especially because they avoid having to set up the command for every test. I also want to see how this looks in the context of the filtering functionality which is in another open PR.

Is this a blocker for you, or would you be ok with me keeping this as is and re-evaluating once the added context of filtering is involved? I definitely agree with your point of view here on indirection, I'm just weighing it against how this might feel once all of the pieces are in place.